main.cc

```diff
@@ -1742,6 +1742,9 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 return ss.local().join_cluster(sys_dist_ks, proxy);
             }).get();
 
+            supervisor::notify("starting tablet split monitor");
+            ss.local().start_tablet_split_monitor();
+
```

Comment:
Either storage_service should start it itself, or it should be a separate service. The way it is written, it's not self-contained and cql_test_env misses it.

Reply:
Ok.